DXRBalance/DeusEx/Classes/AugmentationManager.uc

```diff
@@ -180,6 +184,10 @@ simulated function Float CalcEnergyUse(float deltaTime)
                 bBoosting = true;
             }
             energyUse += f;
+
+            if (anAug.bAutomatic == false && anAug.bAlwaysActive == false) {
```

Comment:
auto augs actually do make sound while they're active though, maybe this should be `if( f > 0 )`

Reply:
also how does this interact with the sound that hazmat suits make

Reply:
A simple check for `(f > 0)` is insufficient because it silences the Light augmentation. Adding that check though seems to make everything work as intended.

Reply:
It has no effect on suits. The vanilla code seems to assume that the player's `AmbientSound` is only for the aug hum. My guess is the hum from a suit is coming out of the suit itself.

Reply:
Indeed it does. From ChargedPickup:

```
function ChargedPickupBegin(DeusExPlayer Player)
{
	Player.AddChargedDisplay(Self);
	PlaySound(ActivateSound, SLOT_None);
	if (LoopSound != None)
		AmbientSound = LoopSound;
```